spec/fpm/package/deb_spec.rb

```diff
@@ -358,13 +358,23 @@ def dpkg_field(field)
 
     it "it should output bit-for-bit identical packages" do
       # Check prerequisites
-      puts("Using " + `which #{tar_cmd}` + `#{tar_cmd} --version`)
-      puts("Exit status of '#{tar_cmd} -cf /dev/null --mtime=@0 --sort=name /dev/null' is #{$?.exitstatus}")
+      # FIXME: ar_cmd and tar_cmd should tell us this
+      puts("Using ar " + `which #{ar_cmd[0]}` + `#{ar_cmd[0]} --version`)
+      FileUtils.rm_f("foo.ar")
+      system("#{ar_cmd.join(' ')} foo.ar /dev/null && env LC_TIME=C TZ=GMT ar tv foo.ar | grep 1970 > /dev/null")
+      FileUtils.rm_f("foo.ar")
+      if $?.exitstatus != 0
+        skip("This system doesn't seem to have an ar that can omit timestamps")
```

Comment:
Seems like this should be in FPM, not the tests. Otherwise a user could request a repeatable-build and think fpm is doing the right thing but the `ar` isn't capable and we end up lying to the user, right?